Jenkinsfile

```diff
@@ -234,30 +234,30 @@ def Tasks = [
     setJavaVersion $java
     npm install &&
     sbtretry 'set scalaJSLinkerConfig in $testSuite.v$v ~= (_.withESFeatures(_.withUseECMAScript2015(false)))' \
-        'set jsEnv in $testSuite.v$v := new NodeJSEnvForcePolyfills()' \
+        'set Seq(jsEnv in $testSuite.v$v := new NodeJSEnvForcePolyfills(), MyScalaJSPlugin.wantSourceMaps in $testSuite.v$v := false)' \
         ++$scala $testSuite$v/test &&
     sbtretry 'set scalaJSLinkerConfig in $testSuite.v$v ~= (_.withESFeatures(_.withUseECMAScript2015(false)))' \
-        'set jsEnv in $testSuite.v$v := new NodeJSEnvForcePolyfills()' \
+        'set Seq(jsEnv in $testSuite.v$v := new NodeJSEnvForcePolyfills(), MyScalaJSPlugin.wantSourceMaps in $testSuite.v$v := false)' \
```

Comment:
That's kind of a new style. Is it different than multiple `set` calls?

(FWIW: I think this needs some love. I have been thinking of introducing a `sjsTestMode` or something and make all the settings dependent on it. This would also make it easier for us to test one of the things. At least I'm tired of copying things out of the `Jenkinsfile` :P).

Reply:
It's the same as multiple `set` calls. I chose this style in this specific case because the `jsEnv` setting must be accompanied by the `wantSourceMaps` setting. It makes it a bit clearer that they must not be separated that way.

I also agree that a general test mode would be nice, but that is probably for a different PR.